exampleCourse/questions/examplesStringInput/question.html

```diff
@@ -66,3 +66,17 @@
 <pl-string-input answers-name="ans5" label="blank input box = " allow-blank="true">
 </pl-string-input>
 </p>
+
+<pl-question-panel>
+<p> 6) To allow for case insensitivity, use <code>ignore-case="true"</code>. 
+    This will autoconvert both the student answer and the correct answer to lower case. </p>
```

Comment:
Can we add an instruction here on how to answer this question? The questions above all do this. For example, we could add “Enter the string below with any choice of uppercase and lowercase letters.”

Reply:
```suggestion
Enter the string below with any choice of uppercase and lowercase letters.
```